app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,15 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.FitLife">
+        <activity
+            android:name=".GroupChallenges"
+            android:exported="false" />
+        <activity
+            android:name=".fitnessMain"
+            android:exported="false" />
+        <activity
+            android:name=".socialMainPage"
+            android:exported="false" />
```

Comment:
Caused an issue for me, have to make sure this is updated so that every page opens up without app crashing.